app/parse.py

```diff
@@ -8,9 +14,59 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_list_tags(quote_soup: BeautifulSoup) -> list[str]:
+    tags_soup = quote_soup.select(".tags > .tag")
+    tags = [tag.text for tag in tags_soup]
+
+    return tags
+
+
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_soup.select_one(".text").text,
+        author=quote_soup.select_one(".author").text,
+        tags=parse_list_tags(quote_soup)
+    )
+
+
+def get_url_next_page(page_number: str) -> str:
+    url = f"{SCRAPE_HOME_URL}page/{str(page_number)}/"
+    return url
+
+
+def get_single_page_quote(page_soup: BeautifulSoup) -> [Quote]:
+    quotes = page_soup.select(".quote")
+
+    return [parse_single_quote(quote_soup) for quote_soup in quotes]
+
+
+def get_all_quotes() -> list:
+    page_num = 0
+    all_quotes = []
+    while page_num < 11:
```

Comment:
Replace the hardcoded value 11 with a constant, e.g. MAX_PAGE_NUM, to make the code more maintainable.